crates/rpc/src/method/subscribe_events.rs

```diff
@@ -742,16 +890,17 @@ mod tests {
                     .unwrap();
                 }
                 db.commit().unwrap();
+                tracing::debug!("Inserted {} blocks", num_blocks);
```

Comment:
does tracing actually work in tests?

Reply:
Only if you enable it. The easiest way is to just use `#[test_log::test(tokio::test)]` instead of `#[tokio::test]`.